lib/srv/discovery/access_graph.go

```diff
@@ -65,7 +65,7 @@ func (s *Server) reconcileAccessGraph(ctx context.Context, currentTAGResources *
 		// We will send a delete request for all resources and return.
 		upsert, toDel := aws_sync.ReconcileResults(currentTAGResources, &aws_sync.Resources{})
 
-		if err := push(stream, upsert, toDel); err != nil {
+		if err := push(stream, upsert, toDel, []*accessgraphv1alpha.AWSMetadata{}); err != nil {
```

Comment:
just add aws metadata to the resources and let reconciler decidde when we need to push/delete/not push anything